docs/user/metacat/source/themes.rst

```diff
@@ -1,21 +1,18 @@
 Modifying and Creating Themes
 =============================
-.. versionadded:: 2.2.0
 
 .. contents::
 
-|
-
 Metacat's theming system, `MetacatUI <https://github.com/NCEAS/metacatui>`_, is deployed
 separately from Metacat, allowing more independent user interface customization. Check the
```

Comment:
Sorry @artntek, do you mean to move the screenshot above the second paragraph?

Reply:
no - I meant replace:

> "allowing more independent user interface customization"

with:

> "allowing more independent visual customization"

...to avoid having "user interface" twice in the same sentence